### PR TITLE
Fix broken cli tests

### PR DESCRIPTION
**Pull request by BenoitZugmeyer**
**What**: The CLI tests are failing since #54.

**Why**: #54 changes the output of files printed to stdout, but didn't change the expected output in the tests.

**How**: Fixed by adjusting the expected output + adding CLI tests to the CI.

### CODE REVIEW

File: package-scripts.js [hunk 1]
```diff
@@ -49,7 +53,7 @@ module.exports = {
     },
     validate: {
       description: 'This runs several scripts to make sure things look good before committing or on clean install',
-      script: concurrent.nps('lint', 'build', 'test'),
+      script: concurrent.nps('lint', 'build', 'test', 'test.cli'),
```
Comment (kentcdodds) [inline package-scripts.js]: Oh! Nice! I had no idea we weren't running those! 😅 thank you!

**Comment by kentcdodds** [comment]
Hmmm... Looks like the tests don't pass on travis.

**Comment by BenoitZugmeyer** [comment]
Sorry I edited the first commit to make sure the prettier-eslint output isn't printing colors. But it seems it still doesn't work on travis... I'm on it.

**Comment by BenoitZugmeyer** [comment]
Finally! I really hate `chalk`. Please check again the first commit, if it seems OK to you.

**Comment by kentcdodds** [comment]
Thank you very much!

**Pull request merged**